packages/flutter_tools/lib/src/flutter_cache.dart

```diff
@@ -216,18 +216,18 @@ class FlutterSdk extends EngineCachedArtifact {
 
   @override
   List<List<String>> getBinaryDirs() {
-    // Currently only Linux supports both arm64 and x64.
+    // Linux and Windows both support arm64 and x64.
```

Comment:
@cbracken Doesn't macOS support ARM64 too with Apple silicon? Lines 227 and 232 below manually specify `x64` instead of using `$arch`, is that intended?